browser/ui/tabs/split_view_tab_strip_model_adapter.cc

```diff
@@ -78,6 +78,27 @@ bool SplitViewTabStripModelAdapter::SynchronizeGroupedState(
   return true;
 }
 
+bool SplitViewTabStripModelAdapter::SynchronizePinnedState(
+    const SplitViewBrowserData::Tile& tile,
+    const tabs::TabHandle& source) {
+  auto [tab1, other_tab] = tile;
```

Comment:
nit
```suggestion
  auto [tab1, tab2] = tile;
```
for consistency